cwms-data-api/src/main/java/cwms/cda/data/dto/Project.java

```diff
@@ -0,0 +1,317 @@
+package cwms.cda.data.dto;
+
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+import cwms.cda.api.errors.FieldException;
+import cwms.cda.formatters.Formats;
+import cwms.cda.formatters.annotations.FormattableWith;
+import cwms.cda.formatters.json.JsonV2;
+import java.time.Instant;
+
+@JsonDeserialize(builder = Project.Builder.class)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonNaming(PropertyNamingStrategies.KebabCaseStrategy.class)
+@FormattableWith(contentType = Formats.JSON, formatter = JsonV2.class)
+public class Project extends CwmsDTO {
+
+    private final String name;
+    private final Double federalCost;
+    private final Double nonFederalCost;
+    private final Instant costYear;
+    private final String costUnit;
+    private final Double federalOAndMCost;
+    private final Double nonFederalOAndMCost;
+    private final String authorizingLaw;
+    private final String projectOwner;
+    private final String hydropowerDesc;
+    private final String sedimentationDesc;
+    private final String downstreamUrbanDesc;
+    private final String bankFullCapacityDesc;
+    private final String pumpBackLocationId;
+    private final String pumpBackOfficeId;
+    private final String nearGageLocationId;
```

Comment:
Same with nearGage - should this just be a Location object?   Or should I make a LocationTemplate/LocationRef class that is just office and id and use that ?

Reply:
Oh, I like that that idea. You may not always need the full Location data, but something organized and easier to search with is a good idea.